fsql.go

```diff
@@ -13,10 +13,12 @@ import (
 const Version = "0.2.1"
 
 var q *query.Query
-var attrs = [4]string{"mode", "size", "time", "name"}
 
-// output prints the result value for each SELECTed attribute. Order is based
-// on the order the attributes appear in attrs.
+// All valid attributes. Not that order of output is based on the order that
+// attributes appear in this array.
```

Comment:
This comment is kind of out of place given it's sort of in reference to my misguided comment. Probably just "All valid attributes" works.